src/components/menu.svelte

```diff
@@ -33,6 +70,34 @@
     <h1>Mega Resume</h1>
   </div>
 
+  <AddEntryButton
+    text={'Open resume'}
+    click={() => {
+      const fileUpload = document.getElementById('file-upload');
+      if (fileUpload != null) fileUpload.click();
+      return null;
+    }}
+  />
+
+  <AddEntryButton
+    text={'Save resume'}
+    click={() => {
+      saveDataToJSONFile();
+      return null;
+    }}
+  />
+
+  <AddEntryButton
```

Comment:
Is there a way to hide this in the distribution version of the site? Can investigate later.